src/components/data/Scroll.js

```diff
@@ -16,7 +16,7 @@ import { downcastRef } from '../../util/typing';
  *
  * @param {CommonProps & ScrollProps & HTMLAttributes} props
  */
-export default function Scroll({
+const ScrollNext = function Scroll({
```

Comment:
Differentiating between the export name (which must be unique at this time to prevent `rollup` collisions) and `function.name` (which is used by tests in this package, and sets the display name — handy for app-side tests).